homeassistant/components/rituals_perfume_genie/__init__.py

```diff
@@ -26,12 +29,46 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Rituals Perfume Genie from a config entry."""
-    session = async_get_clientsession(hass)
-    account = Account(session=session, account_hash=entry.data[ACCOUNT_HASH])
+    _LOGGER.debug("async_setup_entry start (entry_id=%s)", entry.entry_id)
+    session = async_create_clientsession(hass)
```

Comment:
Why do we change from `async_get_clientsession` to `async_create_clientsession`?
(I'm not saying it's wrong, I just want to understand the change)